sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -1372,7 +1372,7 @@ class Std(private val additionalNativeFunctions: Map[String, Val.Builtin] = Map.
       v match{
```

Comment:
I think we need `v.force` here?

Reply:
Oh, hmm, I guess that this shorthand `builtin` syntax is still continuing to force the input arguments.